_posts/2023-06-15-install-tutorial.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Installing CloudForest"
+date: 15-June-2023
+categories:
+  - Getting Started
+tags:
+  - Jekyll
+  - update
+---
+
+##### 1. Install and run Docker
+Installation directions vary by operating system. Detailed instructions can be found [here](https://docs.docker.com/get-docker/). Once downloaded and run, Docker runs in the background.
+
+***NOTE:*** *If using Windows, it is recommended to download and install a ***WSL 2*** linux instance on your computer for easier integration with Docker upon install. More details on installing ***WSL 2*** can be found [here](https://www.omgubuntu.co.uk/how-to-install-wsl2-on-windows-10).*
+
+##### 2. Install CloudForest
+Installers are located in the following repository: https://github.com/TreeScaper/SSBWorkshop_2023.
+
+###### (MacOS)
+1. Download **launchers/cloudforest_launchor_macos_v0.0.x.zip** from the workshop repository.
+2. Double click to extract into Downloads folder.
+3. Right-click CloudForest.command file and select Open.

```

Comment:
Control-click ?

Reply:
Sure, I can say "Right-click or control-click" if that works.